auth/login.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>login</title>
+    <link rel="stylesheet" href="../utilities.css">
+    <link rel="stylesheet" href="../auth/auth.css">
+</head>
+
+<body>
+    <div class="ecom-navbar">
+        <a href="../index.html">
+            <div class="ecom-title landing-page-name">Adore</div>
+        </a>
+
+        <div class="ecom-login-navpills">
+
+            <div class="ecom-navpill ecom-login-navbar">
+                <i class="fas fa-2x fa-shopping-cart ecom-cart"></i>
+            </div>
+            <div class="ecom-navpill ecom-login-navbar">
+                <i class="fal fa-2x fa-heart ecom-wishlist"></i>
+            </div>
+            <div class="ecom-navpill ecom-login-navbar">
+                <i class="fas fa-2x fa-user-alt ecom-profile"></i>
+            </div>
+
+        </div>
+    </div>
+    <div class="login-box">
+        <div class="form-box">
+            <h2 class="login-title">Login</h2>
+            <form>
+                <div class="input-box">
+                    <span class="user-data">Username</span>
+                    <input type="text" name="">
+                </div>
+                <div class="input-box">
+                    <span class="user-data">Password</span>
+                    <input type="password" name="">
+                </div>
+                <div class="remember">
+                    <label>
+                        <input type="checkbox" name="">
```

Comment:
In inputs, you can include placeholders as well for better User experience:)

Reply:
sure done!